qpdf will fail pkgconf test pipeline

If the pkgconf test pipeline is added to qpdf it will fail in the following way:

```
2024/11/12 23:14:57 INFO running step "test/pkgconf"
2024/11/12 23:14:57 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:14:57 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:14:57 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:14:57 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:14:57 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:57 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:57 WARN + basename /home/build/melange-out/qpdf uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:57 WARN + dev_pkg=qpdf uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:57 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:57 WARN + apk info -L qpdf uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:57 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:57 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:57 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:57 WARN + grep -q ^Name: usr/lib64/pkgconfig/libqpdf.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:57 WARN + basename usr/lib64/pkgconfig/libqpdf.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:57 WARN + lib_name=libqpdf uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:57 WARN + echo usr/lib64/pkgconfig/libqpdf.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:57 WARN + grep -q '^usr/lib/pkgconfig/libqpdf.pc$\|^usr/share/pkgconfig/libqpdf.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:57 INFO pod 40898196ae1d5dc4e3a8beb46a98cefb29953d9ba588847982b51651dbe65134 terminated
2024/11/12 23:14:57 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:14:57 INFO   workspace dir: /home/user/tmp/melange-workspace-4241030744
2024/11/12 23:14:57 INFO   guest dir: /home/user/tmp/melange-guest-2414094553
2024/11/12 23:14:57 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/qpdf] Error 1
```

--- a/qpdf.yaml
+++ b/qpdf.yaml
@@ -1,7 +1,7 @@
 package:
   name: qpdf
   version: 11.9.1
-  epoch: 3
+  epoch: 4
   description: Command-line tools and library for transforming PDF files
   copyright:
     - license: Apache-2.0
@@ -32,6 +32,7 @@ pipeline:
     runs: |
       cmake -B build -G Ninja \
       	-DCMAKE_INSTALL_PREFIX=/usr \
+      	-DCMAKE_INSTALL_LIBDIR=/usr/lib \
       	-DCMAKE_BUILD_TYPE=MinSizeRel \
       	-DBUILD_STATIC_LIBS=OFF \
       	-DBUILD_DOC_PDF=OFF \
@@ -52,6 +53,9 @@ subpackages:
       runtime:
         - qpdf
     description: qpdf dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: qpdf-doc
     pipeline:

